src/file-map-url-loader.ts

```diff
@@ -23,21 +23,31 @@ import {parseUrl} from 'polymer-analyzer/lib/utils';
 export class FileMapUrlLoader implements UrlLoader {
```

Comment:
The Analyzer's one is similar but I preferred to use this loader because I could preserve the File objects and not have to `toString()` all of their buffer contents as I loaded the map.  A subsequent PR could look to unify behind `InMemoryOverlayUrlLoader` but I didn't think it bought much to overhaul it.  I did actually start down the path of doing so, but aborted.